.github/workflows/main.yml

```diff
@@ -0,0 +1,39 @@
+name: UpCloud php api test
+on:
+  push:
+    branches:
+      - master
+  pull_request:
+    branches:
+      - master
+jobs:
+  Build:
+    strategy:
+      matrix:
+        os-php-versions: [ 
+          { os: ubuntu-16.04, php: 5.6 },
+          { os: ubuntu-16.04, php: 7.0 },
```

Comment:
point to latest as with others?

Reply:
Latest is from 7.1 upwards https://github.com/marketplace/actions/setup-php-action